### ISSUE TITLE
Btn not returning to original state on touch platforms

### ISSUE THREAD

**Issue by spectrolite** [quasarframework/quasar#756]
### Software version

Quasar: 0.14-beta
OS: Android 6.0.1
Browsers: Chrome 59 and beta 

### What did you get as the error?
Most buttons get stuck in the hover state at the end of the ripple (instead of going back to normal). Clicking anything else restores the correct style. Buttons that trigger a modal or give focus to some other element are not affected. But most simple buttons and possibly some configurations of fab or other quasar component containing a button may be affected.

### What were you expecting?
The button returns to its original state after the ripple ends.

### What steps did you take, to get the error?
Touch a simple button (which has a dark background color, otherwise it's barely noticeable)
e.g. 
```
<q-btn color="deep-purple-9" icon="gps_fixed">
  Test
</q-btn>
<q-btn color="deep-purple-9" icon="gps_fixed">
  Reference
</q-btn>
```

**Comment by rstoenescu** [comment]
Hmm, could not reproduce in my tests. Will investigate. Thx for reporting.

**Comment by spectrolite** [comment]
sidenote: all buttons I’ve tested had `@click`and `small` attributes defined.

**Comment by spectrolite** [comment]
It also happens to the slider button, and basically any button that is not followed by a change in focus.

**Comment by aldanor** [comment]
Happens to me as well, in both mat and ios themes.

In my example it's something like this:

```html
<q-btn big flat icon="done" color="positive" />
```

After tappping the button (iPhone, iOS 10), the button doesn't return to original state.

**Comment by rstoenescu** [comment]
Hi, any gif available to highlight the issue pls?

**Comment by aldanor** [comment]
Looks like it also happens in the official examples in the docs -- the easiest way to see it is tapping the "outline" buttons.

**Comment by aldanor** [comment]
I'm not sure how to record a gif since it's a phone and not an emulator :) Basically, after tap the button is stuck in hover state. You then have to tap elsewhere in the body to reset it back.

**Comment by aldanor** [comment]
For the sake of completeness, I think this also happens to clickable QListItem on mobile. (gets stuck in hover until you tap on something else)

**Comment by aldanor** [comment]
![quasar-btn](https://user-images.githubusercontent.com/2418513/29695278-cf6c85ba-8939-11e7-8740-0de278402277.gif)

(ios emulator)

**Comment by rstoenescu** [comment]
Thanks for the additional information. Fix available in edge and future v0.14.2.

**Comment by aldanor** [comment]
@rstoenescu Thanks! Wonder if this could also be fixed for `QListItem`? (it's the same kind of problem)
